stix2/sources/filesystem.py

```diff
@@ -78,45 +83,41 @@ def _check_path_and_write(stix_dir, stix_obj):
                 os.makedirs(os.path.dirname(path))
 
             with open(path, "w") as f:
-                # Bundle() can take dict or STIX obj as argument
-                f.write(str(Bundle(stix_obj)))
+                f.write(str(stix_obj))
```

Comment:
Does this mean the behavior is now changed to not save a Bundle in each file? Can we add a parameter to the FileSystemSink that retains the current behavior (by essentially doing `if self.bundlify: stix_obj = Bundle(stix_obj)`?) It shouldn't be the default, and we don't need to worry about passing the parameter through via higher levels, but we should at least allow an option for the current behavior. 

Also, `_check_path_and_write` doesn't need to be a nested function, and without the bundlify option, it doesn't even need to be inside this class.

Reply:
Yes, it won't save as a bundle. I'll add that option just to the FileSystemSink.

Reply:
Yep, adding to the Store makes sense.